src/dsp_tools/schemas/project.json

```diff
@@ -367,7 +370,7 @@
                     "if": {
                         "properties": {
                             "super": {
-                                "const": "hasLinkTo"
+                                "const": ["hasLinkTo"]
```

Comment:
We have a problem with this line. Even though I have defined "hasValue" as default on lines 85 and 288, the autocompletion in  VS Code fills in "hasLinkTo" if I type "super". 
Apparently, VS Code takes the value that stands in the first if-condition in the "allOf", instead of the default.

Reply:
that's annoying, but nothing we can change. (you could open an issue on the vscode json integration repo, if you feel like it)